packages/commerce-sdk-react/README.md

```diff
@@ -112,6 +112,44 @@ _💡 This section assumes you have read and completed the [Authorization for Sh
 
 To help reduce boilerplate code for managing shopper authentication, by default, this library automatically initializes shopper session and manages the tokens for developers. Currently, the library supports the [Public Client login flow](https://developer.salesforce.com/docs/commerce/commerce-api/guide/slas-public-client.html).
 
+Commerce-react-sdk supports both public and private flow of the [Authorization for Shopper APIs](https://developer.salesforce.com/docs/commerce/commerce-api/guide/authorization-for-shopper-apis.html) guide._
+You can choose to use either public or private slas to login. By default, public flow is enabled.
+
+#### How private SLAS works
+This section assumes you read and understand how private SLAS flow works
+
+To enable private slas flow, you need to pass a PLACEHOLDER value into CommerceApiProvider and keep your **real** value secured and safe.
+Since commerce-sdk-react run isomorphically, you should not pass **real** secret into the provider because it will be exposed on client-side.
```

Comment:
I don't think this package is the correct place to be talking about placeholders. I think we simply need to warn the developer not to use private client unless they have a mechanism to safeguard their secret was it will be available on the browser. 

We can leave the details of using placeholders to the template.